cmd/juju/storage/show.go

```diff
@@ -23,6 +23,8 @@ options:
    juju environment to operate in
 -o, --output (= "")
    specify an output file
+--format (= yaml)
+   specify output format (json|yaml)
```

Comment:
ATM, it wouldn't. Cmd help is buggy and needs to be addressed. I have put a card on board.
Currently, if you run `juju help storage` and `juju help storage show`, you will only get `juju help storage`...
I think that this wording (and other CLI commands that are structured under SuperCommands) will need to be re-worded..